app/governance/page.tsx

```diff
@@ -1,5 +1,57 @@
-import AnimatedBackground from "@/components/animated_background/animatedBackground";
+"use client";
+
+import { useMemo } from "react";
+import { Proposal } from "@/hooks/gov/interfaces/proposal";
+import useProposals from "@/hooks/gov/useProposals";
+import ProposalTable from "./components/ProposalTable/ProposalTable";
+import styles from "./gov.module.scss";
+import Text from "@/components/text";
+import Spacer from "@/components/layout/spacer";
+import Button from "@/components/button/button";
+import useCantoSigner from "@/hooks/helpers/useCantoSigner";
+import Splash from "@/components/splash/splash";
+import { useRouter } from "next/navigation";
 
 export default function GovernancePage() {
-  return <div>{/* <AnimatedBackground /> */}</div>;
+  const router = useRouter();
+
+  const { chainId } = useCantoSigner();
+  const { proposals, isProposalsLoading } = useProposals({ chainId: chainId });
+
+  const sorted_proposals = useMemo(
+    () =>
+      proposals.sort(
+        (a: Proposal, b: Proposal) => b.proposal_id - a.proposal_id
+      ),
+    [proposals.length]
+  );
+
+  return isProposalsLoading ? (
+    <Splash />
```

Comment:
this is great. Copy this format to the staking page for the return like this on loading